docsource/modules150-160.rst

```diff
@@ -6,7 +6,7 @@ Module coverage 15.0 -> 16.0
 +-------------------------------------------------+----------------------+-------------------------------------------------+
 | Module                                          | Status               + Extra Information                               |
 +=================================================+======================+=================================================+
-| account                                         |                      |                                                 |
```

Comment:
Partial everybody :v

Reply:
Todo: a dynamic way to fill `anaylytic_distribution` on `account.move.line`, `sale.order.line` for short is any model that inherit from `analytic.mixin`